contributor-guide/modules/ROOT/pages/design-guide/headers.adoc

```diff
@@ -4,57 +4,40 @@
 
 @anarthal: this is taken verbatim from old content. It may be outdated. Conversion to asciidoc is not complete.
```

Comment:
We should replace all these `anarthal` comments with small admonitions since the page is already live.

Reply:
Agreed, move all the comments to a separate doc, replaced with short sentence as appropriate:
*Content under construction.*
*Legacy content, currently being updated.*